### DISCUSSION TITLE
Change "no data" message in Table Panel

### DISCUSSION THREAD

**Discussion by aha-sage** [grafana/grafana#44611] [Dashboards, Panels & Visualizations]
We are using Grafana to show some information into Table Panels, but, the WHERE condition is all about timing. 
So, we are trying to Change the "No Data" message in Overrides, with no success, because it seems to not be possible when the whole query is empty and not just a column.

See this:
https://github.com/grafana/grafana/issues/28458

So would be great to create an override possibility when there is no data to show but when another message would bring more information than simply "no data".

Thank you!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
